cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -274,11 +281,6 @@ func KnownControllers() []string {
 
 	ret.Insert(
```

Comment:
move the comment about special controllers here, and give guidance if people should not generally add anything else to this list